configuration/pih/scripts/global/ces.js

```diff
@@ -27,11 +27,11 @@ function setUpCholesterolSection() {
       dmSection.hide();
     }
   };
-  $(dmCheckbox).change(function () {
```

Comment:
"$" isn't necessarily available in the global context, but jq is...  I went through and did a global search-and-replace on all our forms...